<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreGUI::SystemOpenFileSettings</class>
 <widget class="QWidget" name="CoreGUI::SystemOpenFileSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Open</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Open .kum file by system in:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
